CHANGELOG.md

```diff
@@ -20,7 +28,7 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
   ([Issue](https://github.com/hashicorp/boundary/issues/830))
   ([PR](https://github.com/hashicorp/boundary/pull/846))
   
-## 0.1.3 (2020/12/18)
+## v0.1.3 (2020/12/18)
```

Comment:
Good catch on this not being consistent, but we should actually do the inverse: remove the `v` prefix on the earlier entries. All of the other HC tools do not use a `v` prefix in the Changelog. (They also use more verbose dates but I felt like this was a nicer paradigm.)